specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2023-04-01-preview/dataprotection.json

```diff
@@ -0,0 +1,7956 @@
+                "type": "integer",
+                "format": "int32"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Validate Cross Region Restore": {
+            "$ref": "./examples/CrossRegionRestore/ValidateCrossRegionRestore.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/locations/{location}/fetchCrossRegionRestoreJob": {
+      "post": {
+        "tags": [
+          "DppJob"
+        ],
+        "operationId": "GetCrossRegionRestoreJob",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Request body for operation",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CrossRegionRestoreJobRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Details about a Cross Region Restore Job",
+            "schema": {
+              "$ref": "#/definitions/AzureBackupJobResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Cross Region Restore Job": {
+            "$ref": "./examples/CrossRegionRestore/FetchCrossRegionRestoreJob.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/locations/{location}/fetchCrossRegionRestoreJobs": {
+      "post": {
+        "tags": [
+          "DppJob"
+        ],
+        "operationId": "ListCrossRegionRestoreJobs",
```

Comment:
rename `ListCrossRegionRestoreJobs` to `CrossRegionRestoreJobs_List`

Reply:
Done